builds/android/app/src/main/java/org/easyrpg/player/game_browser/GameScanner.java

```diff
@@ -94,8 +94,20 @@ private void scanGames(){
             first_directory = false;
         }
 
-        // Sort the games list
-        Collections.sort(gameList);
+        // Sort the games list : alphabetically ordered, favorite in first
+        List<GameInformation> favoriteGameList = new ArrayList<GameInformation>();
+        List<GameInformation> noFavoriteGameList = new ArrayList<GameInformation>();
+        for(GameInformation game : gameList) {
+            if(game.isFavorite()) {
+                favoriteGameList.add(game);
+            } else {
+                noFavoriteGameList.add(game);
+            }
+        }
+        Collections.sort(favoriteGameList);
```

Comment:
I'm also a bit rusty. Havn't used Java since Java 6. Here is the solution in Java 8 using the new "comparing" API that accepts Functions (omg, you can pass _functions_ as arguments, _finally_)
```java
gameList.sort(
  Comparator.comparing(GameInformation::isFavorite, Comparator.reverseOrder()).
  thenComparing(GameInformation::getTitle));
```